packages/ocular/src/services/api-auth.ts

```diff
@@ -0,0 +1,214 @@
+import { EntityManager } from "typeorm";
```

Comment:
Why do we need this file? This looks like a copy of OAuthService.

Reply:
I deleted that file , I don't know why it is still there , will delete it again , no use of this file